### PR TITLE
[WIP] Convert create_function calls to true anonymous functions

### PR DESCRIPTION
**Pull request by Dragooon**
With the recent bump in minimum PHP requirement as PHP 5.3, we can use closures and true anonymous functions, which are better, more readable and less memory intensive.

Currently I believe I got almost all of them but a few are still left, and these are barely tested. So don't merge them yet and any help testing them will be appreciated. Thanks!

### CODE REVIEW

File: Sources/Groups.php [hunk 1]
```diff
@@ -93,34 +93,35 @@ function GroupList()
 					'value' => $txt['name'],
 				),
 				'data' => array(
-					'function' => create_function('$rowData', '
+					'function' => function ($rowData)
+					{
 						global $scripturl;
```
Comment (live627) [inline Sources/Groups.php]: Can't this use the 'use' keyword?

Comment (Dragooon) [inline Sources/Groups.php]: For $scripturl? It can, but it was this way almost all the places so it'd be required at all the places
/me is lazy

**Comment by Dragooon** [comment]
The last commit will make @live627 happy :D

**Comment by Dragooon** [comment]
Uhh...I think I kinda messed up this PR :P

**Pull request closed without merging**